ophyd/controls/positioner.py

```diff
@@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
                                         name=self.name))
 
             self._readback.subscribe(self._pos_changed)
-
-            self._set_position(self._readback.value)
```

Comment:
Re: @klauer's initial comment about `motor.position` doing a connectivity check, is this a good application for a `ensure_connection` decorator, as @dchabot originally suggested in conversation?